cimr/prompt.py

```diff
@@ -89,27 +97,70 @@ def add_subparser_processor(subparsers):
         action='store_true',
         help='integration of the input data into the data repository',
     )
+    pargs.add_argument(
+        '--query',
+        default=False,
+        dest='query',
+        action='store_true',
+        help='pull down annotations for a list of gene_id\'s',
+    )
 
     # common processor arguments
     parser.add_argument(
-        '--filename',
+        '--file-name',
         default=None,
         type=pathlib.Path,
-        dest='filename',
+        dest='file_name',
         help='file containing summary statistics or annotation data',
     )
     parser.add_argument(
-        '--datatype',
+        '--data-type',
         default=None,
-        dest='datatype',
-        help='currently supported datatypes include \{\'gwas\', \'eqtl\'\}',
+        dest='data_type',
+        help='currently supported data types include: gwas, eqtl, '
+             'tad, and gene',
     )
     parser.add_argument(
         '--genome-build',
         default='b38',
         dest='genome_build',
         help='human genome build used for the input file mapping',
     )
+    parser.add_argument(
+        '--update-map',
+        default=False,
+        dest='update_map',
+        action='store_ture',
+        help='whether to update b37 data to b38 based on provided references',
+    )
+
+    # not required for data-type gwas
+    parser.add_argument(
+        '--cell-type',
+        default=None,
+        type=str,
```

Comment:
This is optional, because `str` is the default type, but feel free to keep it if you like.